### ISSUE TITLE
Default video size is stretched too far

### ISSUE THREAD

**Issue by JXH2014** [georgepaterson/jquery-videobackground#73]
First of all, great work on the project!

It seems to me that the default video is resized up to 1400px (depending on the screen resolution).

E.g. I have a 720p video, For testing purposes, if I resize my browser to 720p resolution (1280px x 720px), the video should fit neatly in there. But it's still stretched to the right side for some reason (on my monitor, an extra 200px or so, making the video off-center). 

In other words, in order to see the full video, I have to stretch my browser further than it currently is (in windowed mode). On a 1080p screen I run with a browser window of around 1300px, but the video is stretched beyond that, for some reason.

Hope this makes sense!

**Comment by georgepaterson** [comment]
Do you have an example i can look at?

There isn't a default size set, the plugin just places the video in the container you've set it, then proportionally resizes using native browser methods.

**Comment by JXH2014** [comment]
I don't have an example, I've just been testing it offline. I've tried it on a Macbook where it plays fine -- it seems that the issue is when a browser window is used that's smaller than the native resolution of the display. I've tried this in Chrome, Firefox, and Safari, running a browser window of various sizes on a 1080p display. The video always seems to stretch further. 

The video doesn't seem to be downsizing properly as well (regardless of browser). 

If you can, try with a higher-res video (e.g. 720p) to see if you can reproduce the issue.

**Comment by JXH2014** [comment]
I figured it out. It was because of the min-height: 100% value. It stretches the video to fill the height of the browser, which means it'll stretch the width equally. 

My bad.

**Comment by georgepaterson** [comment]
Sorry forgot about that myself.
